MNE/disp3D/3DObjects/brain/brainsurfacetreeitem.cpp

```diff
@@ -239,13 +281,14 @@ void BrainSurfaceTreeItem::onColorInfoOriginOrCurvColorChanged()
             this->setData(data, BrainSurfaceTreeItemRoles::SurfaceCurvatureColorVert);
             this->setData(data, BrainSurfaceTreeItemRoles::SurfaceCurrentColorVert);
 
+            //Emit the new colors which are to be used during rt source loc plotting
             emit colorInfoOriginChanged(arrayNewVertColor);
 
             //Return here because the new colors will be set to the renderable entity in the setData() function with the role BrainSurfaceTreeItemRoles::SurfaceCurrentColorVert
             return;
         }
 
-        if(sColorInfoOrigin == "Color from annotation") {
+        if(m_sColorInfoOrigin == "Color from annotation") {
```

Comment:
is this really a good comperator operation? how about m_sColorInfoOrigin.contains("annotation")?